pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.8

```diff
@@ -30,6 +30,7 @@
 .Op Fl -build-host Va host
 .Op Fl -target-host Va host
 .Op Fl -use-remote-sudo
+.Op Fl -use-remote-sudo-always
```

Comment:
If we really need a new behavior (do we?), maybe fix the inconsistency of "remote" vs "target"?
Not sure I like "target" because of cross compilation terminology, but target does become less relevant to users as compilers and their Nix expressions improve. "Remote" isn't great either, because both machines may be equally remote, as demonstrated by the test.
Also if you have an office with NixOS machines and you make the on site server responsible for deploying updates, arguably the deployer is more remote.

So I think I prefer target?

```suggestion
.Op Fl -target-use-sudo
```

Using `target` as a prefix reinforces nice grouping, and helps with discovery through shell completions.

Reply:
I've removed the `--use-remote-sudo-always` flag, opting to just make this PR an improvement on the existing `--use-remote-sudo` flag